lib/cypress/app_config.rb

```diff
@@ -40,5 +40,12 @@ def self.[]=(key, val)
       # Allow setting app config during app init, before Rails.cache exists
       clear
     end
+
+    def self.load_yml
+      app_config = YAML.load(ERB.new(File.read("#{Rails.root}/config/cypress.yml")).result)
+      names_config = YAML.load(ERB.new(File.read("#{Rails.root}/config/names.yml")).result)
+      app_config['randomization']['names'] = names_config['names']
```

Comment:
Could this be broken out to into its own constant value in application.rb, perhaps with the name NAME_CONSTANTS? Just like CAT1_CONFIG is [here](https://github.com/projectcypress/cypress/blob/master/config/application.rb#L9). It should then just be accessible by saying `NAME_CONSTANTS['names']`. If you can think of a better variable name than NAME_CONSTANTS thats fine too.